src/backend/libs-and-metadata.md

```diff
@@ -0,0 +1,192 @@
+# Libraries and Metadata
+
+When the compiler sees a reference to an extern crate, it needs to load some
```

Comment:
Do you want to say "external crate" instead since `extern crate` is almost never used explicitly?